kola/tests/ignition/partitions.go

```diff
@@ -0,0 +1,78 @@
+// Copyright The Mantle Authors
+// SPDX-License-Identifier: Apache-2.0
+package ignition
+
+import (
+	"github.com/flatcar/mantle/kola/cluster"
+	"github.com/flatcar/mantle/kola/register"
+	"github.com/flatcar/mantle/kola/tests/util"
+	"github.com/flatcar/mantle/platform/conf"
+)
+
+const (
+	userdata = `{
```

Comment:
I think it would be fitting to add this test under `ignition/filesystem.go` and use Butane or CLC for transpilation.

Reply:
moved it and switched to CLC